.github/workflows/verify_library_generation.yaml

```diff
@@ -0,0 +1,59 @@
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+name: verify_library_generation
+jobs:
+  verify_library_generation:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        java: [ 11 ]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-java@v3
+      with:
+        java-version: ${{ matrix.java }}
+        distribution: temurin
+        cache: maven
+    - name: maven install
+      run: |
+        mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip
+    - name: get gapic-generator-java, protobuf, grpc version
+      run: |
+        set -x
+        sudo apt-get update
+        sudo apt-get install libxml2-utils
+        echo "GENERATOR_VERSION=$(xmllint --xpath "string(//*[local-name()='version'])" gapic-generator-java-pom-parent/pom.xml)" >> $GITHUB_ENV
+        echo "PROTOBUF_VERSION=$(xmllint --xpath "string(//*[local-name()='protobuf.version'])" gapic-generator-java-pom-parent/pom.xml)" >> $$GITHUB_ENV
+        echo "GRPC_VERSION=$(xmllint --xpath "string(//*[local-name()='grpc.version'])" gapic-generator-java-pom-parent/pom.xml)" >> $$GITHUB_ENV
+    - name: clone showcase
+      run: |
+        git clone https://github.com/googleapis/gapic-showcase.git
+    - name: checkout a tag in showcase
+      run: |
+        git checkout v0.28.2
```

Comment:
I would combine the two steps. They are tightly-coupled.

Reply:
thanks, combined